assets/scss/components/_horizontal-content.scss

```diff
@@ -70,4 +70,8 @@
 
 .horizontal-form__wrapper {
   background-color: govuk-colour("light-grey");
+  .autocomplete__input {
+    background-color: white;
```

Comment:
we can use the govuk-frontend colour set to be more future proof by specifying `$govuk-body-background-colour`.